sql-cli/tests/test___main__.py

```diff
@@ -127,6 +128,37 @@ def test_generate_invalid(workflow_name, message, initialised_project_with_tests
     assert message in result_stdout
 
 
+@pytest.mark.parametrize(
+    "workflow_name,exit_code",
+    [
+        ("example_basic_transform", 0),
+        ("non_existing", 1),
+        ("cycle", 1),
+        ("empty", 1),
+    ],
+    ids=[
+        "example_basic_transform",
+        "non_existing",
+        "cycle",
+        "empty",
+    ],
+)
+def test_generate_verbose(workflow_name, exit_code, initialised_project_with_tests_workflows):
+    result = runner.invoke(
+        app,
+        [
+            "generate",
+            workflow_name,
+            "--project-dir",
+            initialised_project_with_tests_workflows.directory.as_posix(),
+            "--verbose",
+        ],
+    )
+    assert result.exit_code == exit_code
+    if result.exit_code != 0:
+        assert result.exception
```

Comment:
It is probably worth adding some checks on the output, which is only printed when the verbose mode is enabled

Reply:
Yes, thanks 👍 Will add more assertions.